### ISSUE TITLE
Stone hammer recipe bugged - can't use multiple rocks

### ISSUE THREAD

**Issue by jedibob5** [CleverRaven/Cataclysm-DDA#16297]
As of 0.C-13312-gdff120d, I'm unable to make a stone hammer using only rocks. The rock in the components list appears brown, as if it was my only tool with hammering quality 1, which normally means it's a tool that can't consume itself in a recipe. However, I grabbed multiple rocks, so I should be able to use one as the tool and the other as the component.

**Comment by kd8qdz** [comment]
Unable to reproduce in 0.c-13314-g9b71645.  Need more information.

**Comment by jedibob5** [comment]
On further investigation, it actually seems to be a mod-related issue. The issue still persists on my existing world after I updated, but then I created a new world with all mods off and it worked fine. It has to be one of the official mods, though, as other than a small mod I made to blacklist shocker zombies, I only use official mods.

My hunch is that it has to do with More Survival Tools, but I'm not sure how I'd go about confirming that. I'm also not sure if issues with official mods are even dealt with here. If they aren't, this issue can safely be closed and I can go bring it up elsewhere. Here's a few screencaps of the bug anyway.

![cdda1](https://cloud.githubusercontent.com/assets/13815411/14719191/8125a7e8-07c0-11e6-8b8f-981cff6c26c2.png)
![cdda2](https://cloud.githubusercontent.com/assets/13815411/14719192/844c8e3c-07c0-11e6-890d-c7aa4f06b68c.png)
